tests/ts/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ts",
+  "version": "1.0.0",
+  "description": "ts testing project",
+  "main": "index.ts",
+  "type": "module",
```

Comment:
Ok, so I had to add `"type": "module"` to properly reproduce issues that were reported, any ideas how to fix this example?

Reply:
Solved it, and release a temporary `5.0.2-alpha` release, which works for all my example projects. Do you mind @area73 to have a go with it on your side and let me know if it works?